frontend/src/components/SingleCompanyProductList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { GetProductsById } from './useProducts'
+import ListElement from './listOfProducts/ListElement'
+import ListHeader from './listOfProducts/ListHeader'
+
+const SingleCompanyProductList = () => {
+  const { companysProducts } = GetProductsById()
+  return (
+    <div>
+      <h2
+        style={{
+          padding: '20px',
+        }}
+      >
+        {companysProducts[0]?.owner}
```

Comment:
Ez lehet működik, de nem szép. Jobb lenne a company adatát is külön lekérni és abból megjeleníteni a nevet.